pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/column/DefaultNullValueVirtualColumnProvider.java

```diff
@@ -88,6 +88,11 @@ public ColumnMetadataImpl buildMetadata(VirtualColumnContext context) {
     ColumnMetadataImpl.Builder builder = getColumnMetadataBuilder(context).setCardinality(1).setHasDictionary(true);
     if (context.getFieldSpec().isSingleValueField()) {
       builder.setSorted(true);
+
+      if (context.getFieldSpec().getDefaultNullValue() instanceof Comparable) {
```

Comment:
(minor) Put `context.getFieldSpec().getDefaultNullValue()` into a local variable to be reused

Reply:
Ack